utility/make_private_namespace.py

```diff
@@ -0,0 +1,41 @@
+""" Create a namespace """
+import os
+import subprocess
+import shlex
+from utility import templating
+from ocs.utils import create_oc_resource
+
+
+def create_private_namespace(name, region='us-east-2'):
+    """
+    Create a namespace.
+
+    Args:
+        name (str): Namespace to be created
+        region (str): AWS region.  Defaults to us-east-2
+
+    Returns:
```

Comment:
Instead of returning True/False can we raise an Exception in the event that we are unable to create the namespace? We have a number of custom Exceptions in `ocs/exceptions.py` where you can add a new one if necessary.

Reply:
Using  OCP.exec_oc_cmd simplifies this.  It raises the exceptions that I think are appropriate.